tests/TestCli.cpp

```diff
@@ -1791,6 +1791,7 @@ void TestCli::testInvalidDbFiles()
  */
 void TestCli::testYubiKeyOption()
 {
+#ifdef WITH_XC_YUBIKEY
     if (!YubiKey::instance()->init()) {
         QSKIP("Unable to connect to YubiKey");
     }
```

Comment:
Shouldn't this be handled transparently by our YubiKey test stub?

Reply:
Perhaps? This might be an anomaly with how tests are built in that case.

Reply:
We have this stub class. Either we should use it everywhere or just not compile YubiKey tests at all.

Reply:
I figured out the root issue and fixed a hidden bug along the way.